cmd/embedded-cluster/uninstall.go

```diff
@@ -404,6 +409,16 @@ var resetCommand = &cli.Command{
 			}
 		}
 
+		if c.Bool("reboot") {
+			// this is a bit more graceful that using sync(2) and reboot(2) but
+			// can take a bit longer
+			if _, err := exec.Command("reboot").Output(); err != nil {
+				return err
+			}
+		} else {
+			return fmt.Errorf("Aborting reboot")
+		}
```

Comment:
```suggestion
		}
```

Reply:
as it is right now, this exits with an error unless you pass in the "--reboot" flag

Reply:
Definitely wasn't the plan. I thought I had put the `else` on the wrong `if`, which essentially I had, but it was actually that I took out a prompt for "Continue with reboot" that had been in there. Should be fixed up now.